authors/apps/highlight/serializers.py

```diff
@@ -0,0 +1,25 @@
+"""
+Imports
+"""
+from rest_framework import serializers
+from .models import Highlight
+
+class HighlightSerializer(serializers.ModelSerializer):
+    """
+    Serializing the data received from the models
+    to display in dictionary format
+    """
+    highlighted = serializers.CharField(
+        required=True,
+        error_messages={"required": "Highlighted field is required"})
+    comment = serializers.CharField(
+        required=True,
+        error_messages={"required": "Comment filed is required"})
```

Comment:
Should this field be required? Can't a user just highlight a portion of an article? 🤷🏽‍♀️

Reply:
No, that makes sense. It should not be required. Lemme fix that.